python/README.md

```diff
@@ -65,7 +65,7 @@ For more information see the in-code documentation.
 
 ### Test
 
-In the test folder we have scripts that implement test classes based on unittest. To run all tests use:
+In the test folder we have scripts that implement test classes based on unittest. To run all unittest use:
 
```

Comment:
Can you please add documentation on how to run the fuzz tests?

Reply:
Information added on next parragraph